lib/miq_automation_engine/service_models/miq_ae_service_container_image.rb

```diff
@@ -13,6 +13,7 @@ class MiqAeServiceContainerImage < MiqAeServiceModelBase
     expose :openscap_rule_results,        :association => true
     expose :exposed_ports,                :association => true
     expose :environment_variables,        :association => true
+    expose :image_ref
```

Comment:
Database columns and virtual_columns are automatically exposed as read-only fields on objects.  This should not be needed.  Unless I am missing something I think this can be closed.